Sources/Controllers/2.0/Settings/ImportExport/OAExportItemsSelectionViewController.mm

```diff
@@ -33,9 +33,13 @@
 #import "OAGlobalSettingsItem.h"
 #import "OADestination.h"
 #import "OAPOIHelper.h"
+#import "OATableViewCustomHeaderView.h"
 
 #include <OsmAndCore/ArchiveReader.h>
 
+#define titleWithDescrCellHeight 60.0
```

Comment:
Why is this code here? When you switch a task, please rebase onto master before writing the new code.